lib/services/sqs_queue.rb

```diff
@@ -60,6 +64,24 @@ def region
     arn[:region] || 'us-east-1'
   end
 
+  def stubbed_requests?
+    !!ENV['SQS_STUB_REQUESTS']
+  end
+
+  def aws_config
+    {
+      :region            => region,
+      :logger            => stubbed_requests? ? nil : Logger.new(STDOUT),
+      :access_key_id     => access_key,
+      :secret_access_key => secret_key,
+      :stub_requests    => stubbed_requests?
```

Comment:
Might want to tweak the indent on this line so it's aligned and add a trailing `,` just so more lines can be modified without a diff.